test/docfx.RegressionTest/Normalizer.cs

```diff
@@ -80,13 +80,12 @@ private static string NormalizeJsonFile(string path)
 
         private static void PrettifyJsonLog(string logPath)
         {
-            var logs = File.ReadAllLines(logPath).OrderBy(line => line);
+            var logs = File.ReadAllLines(logPath).Select(line => Regex.Replace(line, ",\"date_time\":.*?Z\"", "")).OrderBy(line => line);
```

Comment:
If in doubt, give it a comparer
```suggestion
            var logs = File.ReadAllLines(logPath).Select(line => Regex.Replace(line, ",\"date_time\":.*?Z\"", "")).OrderBy(line => line, StringComparer.Ordinal);
```

Reply:
right, `Ordinal` should be more secure. I thought both ways are acceptable. Locally tried `Ordinal` against `Regex replace`, no much perf gap.